configure.py

```diff
@@ -1331,6 +1331,20 @@ def main():
   if is_macos():
     environ_cp['TF_NEED_TENSORRT'] = '0'
 
+  if is_ppc64le():
+    # Check if the linker has MMA Dynamic Dispatch support
+    ld_version = run_shell(['ld', '--version']).split()
```

Comment:
Is a raw `ld` necessarily the linker?

Reply:
I don't think we're using `ld` everywhere.

Reply:
I think that's much better, though we have builds where gcc is not the default compiler either.

Reply:
This Dynamic Dispatch feature only works with gcc. So, I updated my patch to check that.